test/btc-transaction.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
Just an idea, we could have all this in a single file called `sample-btc-transactions` or something like that
```
[
 { 
   rawtx: "....",
   decoded: "...."
 },
{ 
   rawtx: "....",
   decoded: "...."
 }
]
```
What do you think?

Reply:
I like the idea, most because it'll reduce all into a single file.

Reply:
Done